build.psm1

```diff
@@ -931,28 +933,57 @@ function Start-PSPackage {
         [string]$Name = "powershell",
 
         # Ubuntu, CentOS, and OS X, and Windows packages are supported
-        [ValidateSet("deb", "osxpkg", "rpm", "msi", "appx")]
-        [string[]]$Types
+        [ValidateSet("deb", "osxpkg", "rpm", "msi", "appx", "zip")]
+        [string[]]$Type,
+
+        # Generate windows downlevel package 
+        [ValidateSet("win81-x64", "win7-x64")]
+        [ValidateScript({$IsWindows})]
+        [string]$WindowsDownLevel
     )
 
+    # Runtime and Configuration settings required by the package
+    ($Runtime, $Configuration) = if ($WindowsDownLevel) {
+        $WindowsDownLevel, "Release"
+    } else {
+        New-PSOptions | ForEach-Object { $_.Runtime, $_.Configuration -replace "Debug", "Release" }
```

Comment:
I'm not following this: `New-PSOptions` is a function that you can pass `-Configuration Release` to (for Windows).